src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -430,6 +450,26 @@ public function showAll()
 			$hrfNew = $GLOBALS['TL_DCA'][$this->strTable]['list']['new']['href'];
 		}
 
+		$strClass = '';
+		$strPicker = '';
+
+		// Add the picker attributes
+		if ($this->strPickerField)
+		{
+			$strClass = ' picker unselectable';
+			$strPicker .= ' id="tl_select"';
+
+			if (isset($GLOBALS['TL_DCA'][$this->strTable]['config']['picker']['insertTag']))
```

Comment:
It is not limited to insert tags; the picker will return any given value.

But the JavaScript additionally supports the `data-inserttag` and `data-callback` attributes; the first one will wrap the given value in an insert tag and the second one will trigger an Ajax callback. Both are optional.